zerver/lib/bulk_create.py

```diff
@@ -16,6 +18,7 @@ def bulk_create_realms(realm_list):
     Realm.objects.bulk_create(realms_to_create)
 
 def bulk_create_users(realms, users_raw, bot_type=None):
+    # type: (Dict[unicode, Realm], Set[Tuple[Any, Any, unicode, bool]], Optional[int]) -> None
```

Comment:
In general we're avoiding using `Any` in the annotations where possible.  It looks like the first two Any arguments in the Tuples above are strings; you can use `text_type` from the `six` module to annotate those (that is a str or a unicode on Python 2, but just a str on Python 3), so you should annotate them as such.